static-assets/components/cstudio-admin/mods/content-type-propsheet/image.js

```diff
@@ -117,66 +117,122 @@ YAHOO.extend(
-                    widthConstrains = _self.WIDTHCONSTRAINS,
-                    heightConstrains = _self.HEIGHTCONSTRAINS;
-                  message = CMgs.format(langBundle, 'constraintsError');
-
-                  if (
-                    widthConstrains &&
-                    originalWidth <= widthConstrains &&
-                    heightConstrains &&
-                    originalHeight <= heightConstrains
-                  ) {
-                    var itemURL = to.fileName;
-                    _self.valueEl.value = itemURL;
-                    _self.value = itemURL;
-                    _self.updateFn(null, _self.valueEl);
-                  } else {
-                    var callback = {
-                      success: function (content) {
-                        var itemURL = content.message.internalName;
-                        _self.valueEl.value = itemURL;
-                        _self.value = itemURL;
-                        _self.updateFn(null, _self.valueEl);
-                      }
-                    };
-
-                    CStudioAuthoring.Operations.cropperImage(
-                      CStudioAuthoringContext.site,
-                      message,
-                      imageData,
-                      widthConstrains,
-                      heightConstrains,
-                      widthConstrains / heightConstrains,
-                      null,
-                      callback
+                const formDef = CStudioAdminConsole.selectedFormDef;
+                const configPath = `/content-types/${CStudioAdminConsole.contentTypeSelected}/config.xml`;
+                const defPath = `/content-types/${CStudioAdminConsole.contentTypeSelected}/form-definition.xml`;
+                CStudioAuthoring.Service.lookupConfigurtion(CStudioAuthoringContext.site, configPath, {
+                  success: function (config) {
+                    CStudioAdminConsole.selectedFormDef.imageThumbnail = to.fileName;
+                    const xmlConfig = CStudioAdminConsole.Tool.ContentTypes.FormDefMain.serializeConfigToXml(
+                      config,
+                      formDef
+                    );
+                    const xmlFormDef =
+                      CStudioAdminConsole.Tool.ContentTypes.FormDefMain.serializeDefinitionToXml(formDef);
+
+                    CrafterCMSNext.rxjs
+                      .forkJoin([
+                        CrafterCMSNext.services.configuration.writeConfiguration(
+                          CStudioAuthoringContext.site,
+                          defPath,
+                          'studio',
+                          xmlFormDef
+                        ),
+                        CrafterCMSNext.services.configuration.writeConfiguration(
+                          CStudioAuthoringContext.site,
+                          configPath,
+                          'studio',
+                          xmlConfig
+                        )
+                      ])
```

Comment:
Not what we discussed. What changed?
Also, Phil is updating the API to read from `form-definition.xml` instead of `config.xml` and we only need to update that one for the preview image api to retrieve the image correctly.